### DISCUSSION TITLE
chromebook recovery utility blocked

### DISCUSSION THREAD

**Discussion by skoopy-star** [3kh0/ext-remover#151] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
the extension is blocked by admin
![image](https://user-images.githubusercontent.com/116655650/198187465-0f17a009-c10d-4a9f-b2b8-08a67979cb26.png)

**Comment by 3kh0** [comment, marked as answer]
Use it on a home computer or any other computer that is not managed

**Reply by skoopy-star** [reply]
even if it isn't a chromebook it will work?

**Reply by 3kh0** [reply]
@skoopy-star yes, it can be windows, Mac, or even linux
